src/lib/seam/connect/route-types.ts

```diff
@@ -535,7 +535,7 @@ export interface Routes {
       client_session: {
         token: string
         client_session_id: string
-        created_at: string
+        created_at: string | Date
```

Comment:
@codetheweb Is this right? All responses must be primitives right?

Reply:
would be good if we could lint against this somehow in seam connect

Reply:
https://github.com/seamapi/seam-connect/pull/3657